recipes/boost/meta.yaml

```diff
@@ -0,0 +1,91 @@
+{% set version = "1.60.0" %}
+{% set filename = "boost_%s.tar.bz2" % version.replace(".", "_") %}
+
+package:
+  name: boost
+  version: {{ version }}
+
+source:
+  fn:  {{ filename }}
+  url: http://sourceforge.net/projects/boost/files/boost/{{ version }}/{{ filename }}
+  md5: 65a840e1a0b13a558ff19eeb2c4f0cbe
+
+build:
+  features:
+    - vc9               # [win and py27]
+    - vc10              # [win and py34]
+    - vc14              # [win and py35]
+
+requirements:
+  build:
+    - gcc               # [linux]
+    - python
+    - icu               # [unix]
```

Comment:
`icu` is critical to Boost, as the issue I referenced above shows. It seems `icu` changes its library version name on each release, so that's why it needs to be pinned with something like:

    - icu 54*

I don't understand Mike concern because (for example) we ping `libpng` in our Qt recipe with

    - libpng      1.6*

to avoid breaking Qt when libpng is updated :-)